docs/src/HowToGuides/BalanceLaws/how_to_define_bcs.md

```diff
@@ -0,0 +1,52 @@
+# How to define boundary conditions
+
+```@meta
+CurrentModule = ClimateMachine.BalanceLaws
+```
+
+## Discontinuous Galerkin (DG) method
+
+Boundary conditions (BCs) are defined _weakly_ in ClimateMachine's
+DG implementation. We do this by defining BCs _through_ the _fluxes_,
+rather than on the prognostic variables themselves. Therefore, BCs
+are enforced by prescribing fluxes (numerical fluxes, in DG terminology)
+that drive the solution at the boundary to satisfy our desired BCs.
+
+One major advantage of this implementation approach is that BCs are
+handled in phase space, so we only need one implementation for all
+time steppers.
+
+In the context of the PDE that `ClimateMachine.jl` solves:
+
+`R(Y, t) = ∂_t Yᵢ + (∇•(F₁(Y) + F₂(Y,G)))ᵢ - (S(Y,G))ᵢ = 0, G = ∇Y`,
+
+the following functions must be defined for a
+[`BalanceLaw`](@ref) to fully define BCs:
+
+```@meta
+CurrentModule = ClimateMachine.DGMethods.NumericalFluxes
+```
+
+| **Method**                                       | Internal/User-facing  |  If present | Calls to user-facing functions |
```

Comment:
It's not immediately clear to me what is meant by "Calls to user-facing functions" and "Internal/User-facing" columns. They may need some clarification/motivation outside the table -- why does the user need this information and how will they use it when defining BCs?

Reply:
We can add more information. The basic idea is that the methods in the far left column, by default, call the methods to the right, which, by default, call methods to the right of them. This means that users can intercept these calls at any point along the call graph, depending on how they want to specify the BCs-- for example by specifying the numerical flux directly, via the far left column method or only the ghost points (via `boundary_state!`).